app/models/subscription.rb

```diff
@@ -44,6 +42,10 @@ def token
     self.class.encryptor.encrypt_and_sign(token_values)
   end
 
+  def unsubscribe
+    update(email: nil, active: false, unsubscribed_on: Time.zone.now)
```

Comment:
If it is a time, it should be called `unsubscribed_at`

Reply:
Damn it! 😂

Reply:
@cpjmcquillan at this stage, should you also set any location search criteria to null? Locations can be postcodes, which can be construed as PII, and so probably shouldn't be retained after a user thinks they have 'deleted' their subscription.